src/implicitdict/__init__.py

```diff
@@ -330,7 +330,7 @@ def __new__(cls, value: str | datetime.timedelta | int | float, reformat: bool =
             reformat: If true, override a provided string with a string representation of the parsed timedelta.
         """
         if isinstance(value, str):
-            dt = datetime.timedelta(seconds=pytimeparse.parse(value))
+            dt = datetime.timedelta(seconds=pytimeparse.parse(value) or 0)
```

Comment:
Shouldn't we raise a `ValueError` here if parsing fails? Outputting a 0 looks inconsistent